File: t/CS/GrayScaleDemo/UI/SensitivityDialog.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace GrayScaleDemo
{
    public partial class SensitivityDialog : Form
    {
        private int _senValue;

        public int SenValue
        {
            get { return _senValue; }
            set { _senValue = value; }
        }

        public SensitivityDialog(int senValue)
        {
            InitializeComponent();

            _senValue = senValue;
        }

        private void SensitivityDialog_Load(object sender, EventArgs e)
        {
            _numValue.Value = _senValue;
        }

        private void _btnOk_Click(object sender, EventArgs e)
        {
             _senValue = (int) _numValue.Value;
        }



    }
}

File: t/VB/GrayScaleDemo/UI/SensitivityDialog.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************

Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports System

Partial Public Class SensitivityDialog
   Inherits Form
   Private _senValue As Integer

   Public Property SenValue() As Integer
      Get
         Return _senValue
      End Get
      Set(value As Integer)
         _senValue = value
      End Set
   End Property

   Public Sub New(senValue As Integer)
      InitializeComponent()

      _senValue = senValue
   End Sub

   Private Sub SensitivityDialog_Load(sender As Object, e As EventArgs) Handles Me.Load
      _numValue.Value = _senValue
   End Sub

   Private Sub _btnOk_Click(sender As Object, e As EventArgs) Handles _btnOk.Click
      _senValue = CInt(_numValue.Value)
   End Sub
End Class
